integration-test/it-runner/src/test/resources/layers-flat.toml

```diff
@@ -0,0 +1,42 @@
+#
+#  Copyright 2020 The ModiTect authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+[maven]
+  remote = false
+[maven.repositories.flat]
```

Comment:
In particular here `maven` doesn't make so much sense, as that's not a Maven repo structure, right?

Reply:
What about `resolution` instead of `maven`?